### PR TITLE
Converts module to ES6 and adds {options: app}

### PR DESCRIPTION
**Pull request by OmgImAlexis**
As the title states. Opened this as a response to https://github.com/Lesterpig/sockpress/issues/21#issuecomment-451892583

### CODE REVIEW

File: lib/routing.js [hunk 1]
```diff
@@ -38,12 +23,12 @@ var Routing = function (app, options) {
...
     }
 
-    if (typeof namespace !== 'string' || typeof route !== 'string' || (typeof callback !== 'function' && !(callback instanceof Route.constructor))) {
+    if (typeof namespace !== 'string' || typeof route !== 'string' || (typeof callback !== 'function' && !(callback instanceof Route().constructor))) {
       throw Error('Bad arguments in sockpress/routing');
     }
 
     // Either a single route or a Router object
-    if(callback instanceof Route.constructor) {
+    if (callback instanceof Route().constructor) {
```
Comment (OmgImAlexis) [inline lib/routing.js]: I think this may need to be changed.

**Comment by Lesterpig** [comment]
Thanks for your awesome work! I'll do my best to review it as soon as possible :+1:

File: lib/session-middleware.js [hunk 2]
```diff
@@ -1,49 +1,45 @@
...
+import cookieParser from 'cookie-parser'
 
 /** PRIVATE **/
-function setInvalidSession(socket) {
-  socket.session = {};
-  socket.session.save = function (callback) {
-    callback = callback || function () { };
-    callback('Session is not initialized. Init it with a classic express request.');
-  };
+const setInvalidSession = (socket) => {
```
Comment (OmgImAlexis) [inline lib/session-middleware.js]: With the arrow functions you can drop the `()` with a single param.
`= socket =>` vs `= (socket) =>`

Comment (Lesterpig) [reply lib/session-middleware.js]: Oh, I did not know this shortcut! But we should keep those parenthesis for clarity ;)

**Comment by Lesterpig** [comment]
Sorry, was quite busy lately! I merge and will publish a new version asap. Thanks again!

**Pull request merged**